scylla/src/transport/topology.rs

```diff
@@ -819,10 +819,16 @@ async fn query_peers(conn: &Arc<Connection>, connect_port: u16) -> Result<Vec<Pe
 
     let translated_peers_futures = untranslated_rows.map(|row_result| async {
         let (source, raw_row) = row_result?;
-        let row = raw_row.into_typed().map_err(|_| {
-            QueryError::ProtocolError("system.peers or system.local has invalid column type")
-        })?;
-        create_peer_from_row(source, row, local_address).await
+        match raw_row.into_typed() {
+            Ok(row) => create_peer_from_row(source, row, local_address).await,
+            Err(err) => {
+                warn!(
+                    "system.peers or system.local has an invalid row, skipping it: {}",
+                    err
```

Comment:
I really would like to see https://github.com/scylladb/scylla-rust-driver/issues/1021 here. IIUC this would most always return `system.peers or system.local has invalid column type`, no? 

Either way, this LGTM as it addresses the skip part.

Reply:
The `warn!` prints a pretty message contained by `err`, so we should get the deepest possible explanation of what went wrong.